x-pack/plugins/apm/public/components/fleet_integration/apm_agents/agent_instructions_accordion.tsx

```diff
@@ -97,96 +107,155 @@ function TutorialConfigAgent({
+                />
+              </>
+            )}
+          </EuiText>
+          <EuiSpacer />
+        </section>
+      );
+    }
+  );
+
+  const manualInstrumentationContent = (
+    <>
+      <EuiSpacer />
+      {stepsElements}
+    </>
+  );
+
   return (
-    <EuiAccordion
+    <StyledEuiAccordion
       id={agentName}
       buttonContent={
         <AccordionButtonContent agentName={agentName} title={title} />
       }
     >
-      <EuiSpacer />
-      {steps.map(
-        (
-          {
-            title: stepTitle,
-            textPre,
-            textPost,
-            customComponentName,
-            commands,
-          },
-          index
-        ) => {
-          const commandBlock = replaceTemplateStrings(
-            Array.isArray(commands) ? commands.join('\n') : commands || '',
-            docLinks
-          );
-          return (
-            <section key={index}>
-              <EuiText>
-                <h4>{stepTitle}</h4>
-              </EuiText>
-              <EuiSpacer size="s" />
-              <EuiText color="subdued" size="s">
-                {textPre && (
-                  <InstructionsContent
-                    markdown={replaceTemplateStrings(textPre, docLinks)}
-                  />
-                )}
-                {commandBlock && (
-                  <>
-                    <EuiSpacer size="s" />
-                    <EuiCodeBlock isCopyable language="bash">
-                      {commandBlock}
-                    </EuiCodeBlock>
-                  </>
-                )}
-                {customComponentName === 'TutorialConfigAgent' && (
-                  <TutorialConfigAgent
-                    variantId={variantId}
-                    apmServerUrl={apmServerUrl}
-                    secretToken={secretToken}
-                  />
-                )}
-                {customComponentName === 'TutorialConfigAgentRumScript' && (
-                  <TutorialConfigAgent
-                    variantId="js_script"
-                    apmServerUrl={apmServerUrl}
-                    secretToken={secretToken}
-                  />
-                )}
-                {textPost && (
+      {AgentRuntimeAttachment ? (
+        <>
+          <EuiSpacer />
+          <EuiTabbedContent
+            tabs={[
+              {
+                id: 'manual-instrumentation',
+                name: 'Manual instrumentation',
+                content: manualInstrumentationContent,
+              },
+              {
+                id: 'auto-attachment',
+                name: (
+                  <EuiFlexGroup
+                    justifyContent="flexStart"
+                    alignItems="baseline"
+                    gutterSize="s"
+                  >
+                    <EuiFlexItem grow={false}>Auto-Attachment</EuiFlexItem>
+                    <EuiFlexItem grow={false}>
+                      <EuiBetaBadge label="BETA" />
```

Comment:
```suggestion
                      <EuiBetaBadge label={i18n.translate('xpack.apm.fleetIntegration.apmAgent.runtimeAttachment.betaBadge.label',
              {
                defaultMessage:
                  'BETA',
              }
            )} tooltipContent={i18n.translate('xpack.apm.fleetIntegration.apmAgent.runtimeAttachment.betaBadge.tooltipContent',
              {
                defaultMessage:
                  'Auto-attachment for Java is not GA. Please help us by reporting any bugs.',
              }
            )}
 />
```

Reply:
Probably the formatting is bust from copy-pasting, but I think we should add a tooltip to the beta badge in the tab. And I suppose the label should also be i18n formatted.